src/shared_utilities/drivers.jl

```diff
@@ -281,6 +298,34 @@ function turbulent_fluxes_at_a_point(
     return (lhf = LH, shf = H, vapor_flux = Ẽ, r_ae = r_ae)
 end
 
+"""
+    surface_fluxes(atmos::CoupledGlobalAtmosphere,
+                    model::AbstractModel,
+                    Y,
+                    p,
+                    t)
+
+Computes the turbulent surface fluxes terms at the ground for a coupled simulation.
+"""
+function ClimaLSM.turbulent_fluxes(
+    atmos::CoupledGlobalAtmosphere,
+    model::AbstractModel,
+    Y,
+    p,
+    t,
+)
+    # coupler has done its thing behind the scenes already
+    model_name = ClimaLSM.name(model)
+    model_cache = getproperty(p, model_name)
+    return (
+        lhf = model_cache.turbulent_fluxes.lhf,
+        shf = model_cache.turbulent_fluxes.shf,
+        vapor_flux = model_cache.turbulent_fluxes.vapor_flux,
+        r_ae = ClimaCore.Fields.similar(model_cache.turbulent_fluxes.lhf),
```

Comment:
not all models store this, and, in particular, they need to store it only in non-coupled cases. clean this up!